File: src/main/java/com/sean/Ticket.java
package com.sean;

public class Ticket {
    String origin;
    String destination;
    int price;

    public String getOrigin() {
        return origin;
    }

    public void setOrigin(String origin) {
        this.origin = origin;
    }

    public String getDestination() {
        return destination;
    }

    public void setDestination(String destination) {
        this.destination = destination;
    }

    public int getPrice() {
        return price;
    }

    public void setPrice(int price) {
        this.price = price;
    }

    public Ticket(String origin, String destination, int price) {
        this.origin = origin;
        this.destination = destination;
        this.price = price;
    }
}

File: src/main/kotlin/com/sean/Ticket.kt
package com.sean.kotlin

import kotlin.random.Random

data class Ticket(val origin : String, val destination : Int, val price : Int)

fun main() {
    var ticket = Ticket("30",40,50)
    val s = "dadadT"
    println(s.validate())
    println((1..100).random())
}
fun String.validate() : Boolean {
    this.length >= 6
    return true
}
fun IntRange.random() : Int {
    val r = Random.nextInt(endInclusive-start) + start
    return r
}
